### PR TITLE
Update ERC-7562: editorial pass for grammar, clarity, and consistency

### PR DESCRIPTION
**Pull request by forshtat**
## Summary
- Line-by-line editorial pass over ERC-7562's prose: grammar, punctuation, and terminology consistency (P2P/Sybil/UserOperations casing, "2nd" -> "second", contractions, hyphenation, etc.)
- Fixes a handful of outright typos: `incrementNonce())` extra paren, "opsSeen valid" -> "opsSeen value" (x2), "clear test" -> "clear text", `opsInclude` -> `opsIncluded`, `OP-13` -> `OP-013` (matching the zero-padded ID convention), and a Definitions list that mis-numbered items 3-8 as "2."
- No rule IDs, constants, formulas, or normative meaning were changed - this is text quality only.
- Two new rules (`LIM-060`, `LIM-070`) to assign real enforceable meaning to previously implicit `MAX_VERIFICATION_GAS` and `PRE_VERIFICATION_OVERHEAD_GAS` limits

### CODE REVIEW

File: ERCS/erc-7562.md [hunk 1]
```diff
@@ -12,63 +12,62 @@ created: 2023-09-01
 
 ## Abstract
 
-This document describes the rules Account Abstraction protocols should follow, during the validation phase of Account Abstraction transactions,
-such as [ERC-4337](./eip-4337) `UserOperation` or RIP-7560 (Native Account Abstraction), which are enforced off-chain by a
-block builder or a standalone bundler, and the rationale behind each one of them.
+This document describes the validation rules that Account Abstraction protocols must follow for transactions such as an [ERC-4337](./eip-4337) `UserOperation`, alongside the rationale for each rule. Block builders and standalone bundlers enforce these rules off-chain.
```
Comment (forshtat) [inline ERCS/erc-7562.md]:
```suggestion
This document describes the validation rules that [ERC-4337](./eip-4337) Account Abstraction protocol participants must follow for user transactions represented as `UserOperation` structs, alongside the rationale for each rule. Block builders and standalone bundlers enforce these rules off-chain.
```

File: ERCS/erc-7562.md [hunk 2]
```diff
@@ -12,63 +12,62 @@ created: 2023-09-01
...
 
-For the actual interfaces of those contract-based accounts see the definitions in ERC-4337 and RIP-7560.
+For the interfaces of these contract-based accounts, see [ERC-4337](./eip-4337).
 
-This documentation uses the terminology "UserOperation" for a transaction created by a smart contract account, and closely follows [ERC-4337](./eip-4337) terminology.
-However, the rules apply to any Account Abstraction framework that uses EVM code to perform transaction validation and makes a distinction between validation (whether the operation is eligible for inclusion on the protocol level) and execution (on-chain execution and gas payment) in a public mempool.
+This document uses the term "UserOperation" for a transaction created by a smart contract account, following ERC-4337 terminology.
+However, the rules apply to any Account Abstraction framework that uses EVM code to perform transaction validation in a public mempool, and that treats validation and execution as distinct: validation determines whether the operation is eligible for inclusion at the protocol level, while execution covers the on-chain execution and gas payment.
```
Comment (forshtat) [inline ERCS/erc-7562.md]:
```suggestion
Notably, many of these rules can also be applicable in any other Account Abstraction framework that uses EVM code to perform transaction validation in a public mempool, and that treats validation and execution as distinct components of a transaction.
```

File: ERCS/erc-7562.md [hunk 3]
```diff
@@ -174,266 +172,249 @@ Because of the reputation `refresh rate`, note that a malicious paymaster can at
...
+    * **[OP-055]** May call `incrementNonce()` from the `sender`.
 * `*CALL` opcodes:
     * **[OP-061]** `CALL` with `value` is forbidden. The only exception is a call to the `EntryPoint` described above.
     * **[OP-062]** Precompiles:
-        * Only allow known accepted precompiles on the network, that do not access anything in the blockchain state or environment.
-        * The core precompiles 0x1 .. 0x11
-        * The RIP-7212 secp256r1 precompile, on networks that accepted it.
+        * Only known, accepted precompiles on the network that do not access anything in the blockchain state or environment are allowed.
+        * The core precompiles `0x1`–`0x11`.
+        * The `P256VERIFY` secp256r1 precompile defined in [EIP-7951](./eip-7951), on networks that have adopted it.
```
Comment (forshtat) [inline ERCS/erc-7562.md]:
```suggestion
        * The `P256VERIFY` secp256r1 precompile defined in [EIP-7951](./eip-7951).
```

File: ERCS/erc-7562.md [hunk 4]
```diff
@@ -174,266 +172,249 @@ Because of the reputation `refresh rate`, note that a malicious paymaster can at
...
-  * **[EREP-061]** A staked factory may also use a utility contract that calls the `CREATE`
-* **[EREP-070]** During bundle creation, if a staked entity reduce its validation gas by more than 10% 
-    compared to the second validation, that entity is throttled, even if the UserOperation itself didn't revert.
-    (as it might affect the gas calculation based on [EIP-7623](./eip-7623.md) )
+  * **[EREP-060]** If the factory is staked, either the factory itself or the sender may use the `CREATE2` and `CREATE` opcodes.
+    The sender is allowed to use `CREATE` with an unstaked factory as well, per OP-032.
+  * **[EREP-061]** A staked factory may also use a utility contract that calls the `CREATE` opcode.
+* **[EREP-070]** During bundle creation, if a staked entity reduces its validation gas by more than 10%
+    compared to the second validation, that entity is throttled, even if the UserOperation itself did not revert
+    (since this might affect the gas calculation defined by [EIP-7623](./eip-7623)).
```
Comment (forshtat) [inline ERCS/erc-7562.md]:
```suggestion
    compared to the second validation, that entity is throttled, even if the UserOperation itself did not revert, since this might affect the gas calculation defined by [EIP-7623](./eip-7623).
```
